aws-concepts/aws-ses.md

```diff
@@ -0,0 +1,98 @@
+---
+title: AWS SES (Simple Email Service) | BlueFox Email
```

Comment:
I would use Amazon SES in the titles... and maybe in descriptions, we can use AWS SES.

It's because its official name is Amazon SES, and more people search for that.